src/OrbitGl/Viewport.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_VIEWPORT_H_
+#define ORBIT_GL_VIEWPORT_H_
+
+#include "CoreMath.h"
+
+namespace orbit_gl {
+
+/**
```

Comment:
Nit: If I am not wrong we avoid block comments and use `//` at all places. However, even if not it should be
```
/*
 *
 */
```

Reply:
I've changed it to //